scripts/zones/Jugner_Forest/mobs/Sappy_Sycamore.lua

```diff
@@ -12,6 +12,7 @@ entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
     mob:addMod(xi.mod.SLEEPRES, 20)
     mob:addMod(xi.mod.BINDRES, 20)
+    mob:addMod(xi.mod.EARTH_RES, 100)
```

Comment:
EARTH_RES was recently renamed to EARTH_MEVA to reflect is actual effect and avoid problems when resistance rank mods get added later.

Reply:
Corrected to EARTH_MEVA